### PR TITLE
Fix #48 : Add Typescript tutorial 

### LINKED ISSUE #48: Add Typescript tutorial
Evaluate if https://github.com/total-typescript/beginners-typescript would be a good typescript tutorial for new contributors and update the [Learning Resources](https://github.com/oppia/oppia-web-developer-docs/blob/develop/Learning-Resources.md) page

### PR DESCRIPTION
**Pull request by MohitGupta121**
Fix #48 : Add Typescript tutorial

### CODE REVIEW

**Comment by MohitGupta121** [comment]
@Ash-2k3 PTAL, Please have initial review after that pass next review to lead. Thanks!

File: Learning-Resources.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ That said, we strongly recommend that you be open to learning new things. If you
 
   Most of our open issues are in the frontend and require at least some knowledge of HTML, AngularJS/Angular and CSS, so knowledge of frontend coding will serve you well if you'd like to contribute to Oppia over the longer term.
 
+  * TypeScript
+    * Checkout the great TypeScript [tutorial](https://www.freecodecamp.org/news/learn-typescript-beginners-guide/)
```
Comment (U8NWXD) [inline Learning-Resources.md]:
```suggestion
    * Check out this TypeScript [tutorial](https://www.freecodecamp.org/news/learn-typescript-beginners-guide/).
```

To stay consistent with the rest of the page

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Learning-Resources.md (hunk 1 above)
Comment (MohitGupta121) [reply Learning-Resources.md]: Done.

**Review by MohitGupta121 — COMMENTED** [review]
@U8NWXD PTAL, changes applied, Thanks!

**Review by Ash-2k3 — APPROVED** [review]
Thanks @MohitGupta121, LGTM.

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**